html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <label for="email">Email</label>
+    <br />
```

Comment:
avoid `br` tags, use divs to wrap the elements you want to break onto additional lines